# Ломать — не строить
try:
    func(None, None)
except ValueError:
    print("Ура! Ошибка!")
